Makefile

```diff
@@ -1,3 +1,6 @@
+PACKAGE_NAME = osbuild-composer
+VERSION = 3
```

Comment:
Current version is 4, can you please fix it?

Reply:
Sure, I parse it from spec file now.